src/components/login_page/login_form/login_form.jsx

```diff
@@ -0,0 +1,53 @@
+import { useState } from "react";
+import { useDispatch } from "react-redux";
+import { loginCheck } from "../../redux/auth/auth_actions";
+import "./login_form.css";
+import LoginFormHeader from "./login_form_header/login_form_header";
+import LoginFormInput from "./login_form_input/login_form_input";
+import SubmitButtonForm from "./submit_button_form/submit_button_form";
+import { loginFormContent } from "../../../data/login_form_content";
+
+export const LoginForm = () => {
+  const dispatch = useDispatch();
+
+  const [credentials, setCredentials] = useState({
+    username: "",
+    password: "",
+  });
+
+  const handleInputChange = (e) => {
+    const { name, value } = e.target;
+    setCredentials((prevState) => ({ ...prevState, [name]: value }));
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log(loginCheck(credentials.username, credentials.password));
+    dispatch(loginCheck(credentials.username, credentials.password));
```

Comment:
and then we can do it so


```suggestion
    const res = dispatch(loginCheck(credentials.username, credentials.password));
    if(...) navigate('/login')
```